TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -920,6 +920,12 @@ for a method marked with the `objc` attribute.
 
 ### NSApplicationMain
 
+> Note:
```

Comment:
NIT:

`Note` seems a little weak for me. Should we consider to use `Warning`/`important` or add a new "Deprecated" in swift-markdown and swift-docc-render

https://github.com/apple/swift-markdown/blob/c211079c200ce2c5f68160bf75ded005b1c945f1/Sources/Markdown/Interpretive%20Nodes/Aside.swift

![Warning Preview locally](https://github.com/apple/swift-book/assets/43724855/3d4a909d-bbfd-404d-be02-f40222bd487f)

I remember I have seen something similar usage on Apple's documentation for deprecation aside. But I can't find it now. Maybe we can follow the same style.

Reply:
We usually reserve *Warning* asides for more severe topics like potential data loss for end users, so using a custom aside title like *Deprecated* makes more sense to me here.

The (default) DocC theme only renders a custom color for *Warning*, though, so I can see why it's tempting to reach for it for visual distinction alone. (API docs rendered from symbol graphs do support the orange *Deprecated* asides.)

Reply:
It's possible to make the aside have the 'Deprecated' label, it'll be gray in this DocC style though, and I don't know of a way to customize that.

Reply:
Changed to "Deprecated" — having it gray seems reasonable to me, since that's the default note styling and it's already distinctive compared to the surrounding text.